Egregious-Eggnog/reactive-nutshell
# Lr aligning messages

# Description

Added message format to make the messages look like a chat box. Also changed some styling in tasks and friends. Just putting up the PR so that if you guys want to merge new styling, you can. Just make sure all functionality on ALL pages works first before you approve PR. 

Fixes # (issue)

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout lr-aligning-messages`
3. Added message format to make the messages look like a chat box. Also changed some styling in tasks and friends. Just putting up the PR so that if you guys want to merge new styling, you can. Just make sure all functionality on ALL pages works first before you approve PR. 



# Checklist:
